src/style.css

```diff
@@ -1 +1,89 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  --background-color: #e5e5e5;
+  --header-color: #fff;
+  --header-height: 60px;
+  --logo-height: 40px;
+  --logo-width: 40px;
+  --link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+  background-color: var(--background-color);
+}
+
+.header {
```

Comment:
sorry for a little discoordination with the previous mentor, but the correct solution is to add padding: 0 50px; to a header instead of adding paddings to your logo and nav. Since header is a wrapper and if you'd like to fit content inside you should add padding to this wrapper (header), not its children

Reply:
But if I add padding: 0 50px; header size will be 50 + 1200 + 50 instead of 1200. In this case logo and nav will be closer to each one. If I understand right.

Reply:
<img width="1440" alt="Screenshot 2023-05-25 at 10 04 30" src="https://github.com/mate-academy/layout_moyo-header/assets/132390251/ccd87f31-8ec7-47e9-a933-44a63f1be821">

Reply:
But it works if I use header with box-sizing: border-box;

<img width="1202" alt="Screenshot 2023-05-25 at 10 23 19" src="https://github.com/mate-academy/layout_moyo-header/assets/132390251/4f4d6058-d172-4ce8-b429-bb95fb611b41">
.header {
  display: flex;
  position: relative;
  align-items: center;
  justify-content: space-between;
  padding: 0 50px;
  box-sizing: border-box;
  background-color: var(--header-color);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
}